src/app.py

```diff
@@ -38,6 +40,13 @@
 # Schedule handshaking every 30s
 scheduler.add_job(func=send_handshakes, trigger=IntervalTrigger(seconds=30), id='sending_handshakes', replace_existing=True)
 
+# Schedule heartbeat every 30s
+time.sleep(3) # Sleep for 3 seconds to allow the satellite to discover its neighbours
+scheduler.add_job(func=send_heartbeat_to_neighbours, trigger=IntervalTrigger(seconds=15), id='sending_heartbeats', replace_existing=True)
+# Addition and removal of neighbours to mimic movement 
+scheduler.add_job(func=manage_neighbours, trigger=IntervalTrigger(seconds=18), id='managing neighbours', replace_existing=True)
```

Comment:
same comment as i left on patrick's this morning, modify these to sleep then schedule on a thread instead of on main to ensure the neighbour finding happens on time